src/pages/{DocsYaml.slug}-tls.jsx

```diff
@@ -93,6 +93,9 @@ const Page = ({ data, location }) => {
   const description = `Step-by-step instructions for operationalizing ${doc.name} TLS certificates on Linux, Docker, or Kubernetes.`;
   const { siteUrl } = site.siteMetadata;
   const url = `${siteUrl}${location.pathname}`;
+  const unfurl = unfurlImages.edges.find(({ node }) => (
+    node.relativePath.includes(doc.slug)
```

Comment:
Could this one be more specific, just in case we ever have one doc name that's contains another? (eg. `kuberenetes` and `kubernetes ingress`)

```suggestion
    node.relativePath === `${doc.slug}/unfurl.png`
```

It looks like the `kubernetes-ingress` one is uploaded in the wrong spot and shouldn't be in the `sections/` dir.